dev/tests/unit/Magento/FunctionalTestFramework/Console/BaseGenerateCommandTest.php

```diff
@@ -182,13 +176,25 @@ public function testSuiteToTestSyntax()
      */
     public function mockHandlers($testArray, $suiteArray)
     {
-        AspectMock::double(TestObjectHandler::class, ['initTestData' => ''])->make();
+        $testObjectHandler = TestObjectHandler::getInstance();
+        $reflection = new \ReflectionClass(TestObjectHandler::class);
+        $reflectionMethod = $reflection->getMethod('initTestData');
+        $reflectionMethod->setAccessible(true);
+        $output = $reflectionMethod->invoke($testObjectHandler);
+        $this->assertEquals('', $output);
+
```

Comment:
This code does not seem to be necessary since we are not stubbing `initTestData`?

This is not a usual way for unit tests, but it does work for eliminating the usage of AspectMock with no framework refactoring. We can use this solution.

Reply:
@jilu1 Hello! I removed this code.